cc_licenses/templates/base.html

```diff
@@ -8,10 +8,7 @@
     <meta name="description" content="{% block meta-description %}{% endblock %}">
     <meta name="author" content="{% block meta-author %}{% endblock %}">
     <meta name="keywords" content="{% block meta-keywords %}{% endblock %}">
-    {% block extra-meta %}{% endblock %}
-    <link rel="stylesheet" type="text/css" media="all" href="{% static 'css/bundle.css' %}?rev={{ REVISION }}">
-    {% block extra-css %}{% endblock %}
-    <script src="{% static 'libs/modernizr.js' %}"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary/css/vocabulary.css">
 </head>
 <body>
     <div class="container">
```

Comment:
How about bundle.js at the bottom of this file?

Reply:
done